src/coreComponents/constitutive/solid/PorousSolid.hpp

```diff
@@ -55,25 +55,37 @@ class PorousSolidUpdates : public CoupledSolidUpdates< SOLID_TYPE, BiotPorosity,
   GEOSX_HOST_DEVICE
   void smallStrainUpdate( localIndex const k,
                           localIndex const q,
+                          real64 const & pressure,
                           real64 const & deltaPressure,
                           real64 const ( &strainIncrement )[6],
-                          real64 ( & stress )[6],
+                          real64 ( & totalStress )[6],
                           real64 & dPorosity_dPressure,
                           real64 & dPorosity_dVolStrain,
-                          real64 & dTotalStress_dPressure,
+                          real64 ( & dTotalStress_dPressure )[6],
                           DiscretizationOps & stiffness ) const
   {
-    m_solidUpdate.smallStrainUpdate( k, q, strainIncrement, stress, stiffness );
+    // Compute effective stress and store in totalStress
+    m_solidUpdate.smallStrainUpdate( k, q, strainIncrement, totalStress, stiffness );
 
     updateBiotCoefficient( k );
 
+    // Compute  total stress
+    real64 const biotCoefficient = m_porosityUpdate.getBiotCoefficient( k );
+    LvArray::tensorOps::symAddIdentity< 3 >( totalStress, -biotCoefficient * ( pressure + deltaPressure ) );
```

Comment:
Is there a reason not to just pass newPressure as the argument to this function, instead of both pressure and deltaPressure?

Reply:
Do you mean the having the following?

```c++
// Compute  total stress
real64 const biotCoefficient = m_porosityUpdate.getBiotCoefficient( k );
real64 const newPressure = pressure + deltaPressure;
LvArray::tensorOps::symAddIdentity< 3 >( totalStress, -biotCoefficient * newPressure );
```

Reply:
I think he means to only give `newPressure` as input to the `smallStrainUpdate` but, we only have the `strainIncrement` so the porosity is update as a function of the strain and the pressure increments which is why we need `deltaPressure`.

Reply:
Sorry, I meant in the argument list at line 58.  We never actually use the two separately here.

Reply:
Ohh, I see, deltaPressure is actually used below.  Nevermind.